src/js/angular/core/services/jwt-auth.service.js

```diff
@@ -88,8 +88,7 @@ angular.module('graphdb.framework.core.services.jwtauth', [
              * @param {boolean} justLoggedIn Indicates that the user just logged in.
              */
             this.getAuthenticatedUserFromBackend = function(noFreeAccessFallback, justLoggedIn) {
-                SecurityRestService.getAuthenticatedUser().
-                success(function(data, status, headers) {
+                SecurityService.getAuthenticatedUser().then(function(data) {
```

Comment:
Beware of these changes. There were few places where the `success/error` handlers were left intentionally due to sertain reasons that I can't recall at the moment but if possible, I'd prefer if we don't switch them here, now

Reply:
I made such changes as little as possible.